packages/account-v2/src/modules/ManualForm/manualFormInputs.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { Fragment } from 'react';
+import { Field, FieldProps } from 'formik';
+import { useSettings } from '@deriv/api';
+import { WalletDatePicker } from '../../components/base/WalletDatePicker';
+import { WalletText } from '../../components/base/WalletText';
+import { WalletTextField } from '../../components/base/WalletTextField';
```

Comment:
Just to confirm, your plan is to replace these with components from deriv-ui right?

Reply:
yes @matin-deriv, the components placed inside `componets/base`, which includes `Dropzone`, `WalletDatePicker`, `WalletDropdown`, `WalletText` and `WalletTextField` will be replaced with components from deriv/ui, once available.
anyway, let me check if any of these components are available in the latest deriv/ui package at this moment and make changes in this PR.